apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -943,19 +943,59 @@ public Task OpenAsReadOnlyAsync(TimestampBound timestampBound = null, Cancellati
             {
                 throw new InvalidOperationException($"{nameof(OpenAsReadOnlyAsync)} should only be called with ${nameof(EnlistInTransaction)} set to true.");
             }
-            Action transactionEnlister = () => EnlistTransaction(transaction, timestampBound ?? TimestampBound.Strong, null);
+            Action transactionEnlister = () => EnlistTransaction(transaction, timestampBound ?? TimestampBound.Strong, transactionId: null, commitdelay: null);
             return OpenAsyncImpl(transactionEnlister, cancellationToken);
         }
 
+        /// <summary>
+        /// Opens the connection within a <see cref="System.Transactions.TransactionScope"/> specifying a commit delay
+        /// value for the associated Cloud Spanner transaction. See <see cref="SpannerTransaction.CommitDelay"/> for
+        /// more information.
+        /// </summary>
+        public void Open(TimeSpan commitDelay)
```

Comment:
I'm slightly nervous that commit delay might only be *one* option that we want to specify like this, and we could end up with a load of overloads in the future. Would some sort of Spanner.Data-level TransactionOptions type be worth introducing?